src/applications/disability-benefits/686/config/form.js

```diff
@@ -69,6 +69,7 @@ const {
   domesticAddress,
   fullName,
   internationalAddressText,
+  genericLocation,
```

Comment:
Should this be removed? I was under the assumption we were getting rid of it moving forward.